src/gt4py/next/ffront/fbuiltins.py

```diff
@@ -201,6 +202,36 @@ def where(
     raise NotImplementedError()
 
 
+@WhereBuiltinFunction
```

Comment:
Move this to experimental (and optionally change this name, e.g.: `combine_at`).